test/integration/fixtures/blocks/core__quote__deprecated-1.serialized.html

```diff
@@ -1,3 +1,6 @@
-<!-- wp:quote -->
-<blockquote class="wp-block-quote"><p>Testing deprecated quote block...</p><cite>...with a caption</cite></blockquote>
+<!-- wp:quote {"className":"blocks-quote-style-1"} -->
+<blockquote class="wp-block-quote blocks-quote-style-1">
+	<p>Testing deprecated quote block...</p>
+	<footer>...with a caption</footer>
```

Comment:
Why did this change at all if it's not changed to the current markup? Why is this all of a sudden using `footer`?